meta-mbl-distro/recipes-core/systemd/systemd-watchdog-conf.bb

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2019 Arm Limited and Contributors. All rights reserved.
+#
+# SPDX-License-Identifier: MIT
+
+SUMMARY = "Configure systemd watchdog management"
+DESCRIPTION = "Configure systemd to manage the hardware watchdog at runtime."
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
+
+SRC_URI = "file://10-watchdog-mbl.conf"
+
+FILES_${PN} += "${sysconfdir}/systemd/system.conf.d/10-watchdog-mbl.conf"
+
+inherit systemd
```

Comment:
```suggestion
```
I suspect that this is not needed since we are not installing any systemd service file.